packages/victory-pie/src/slice.js

```diff
@@ -22,23 +21,45 @@ const getPath = (props) => {
 };
 
 const evaluateProps = (props) => {
-  /* Potential evaluated props are
-    1) style
-    2) radius
-    3) innerRadius
-    4) everything else
-  */
+  /**
+   * Potential evaluated props are
```

Comment:
Is it worth documenting `slice` here (used in `getPath`)?

Reply:
`slice` isn't evaluated, though, since it's not a prop a user is meant to provide. It's like `datum` with some additional layout information added